### DISCUSSION TITLE
Would revalidating a static page work when self-hosted?

### DISCUSSION THREAD

**Discussion by welschmoor** [sveltejs/kit#10964] [General]
Hello,

with NEXT.js revalidating a statically generated page works [out of the box](https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration#self-hosting-isr) when self-hosting.
Can somebody update me on what behavior to expect with Svelte Kit in this regard? Thanks!

**Comment by teemingc** [comment]
Incremental Static Regeneration (ISR) is currently only supported when using the Vercel adapter and deploying to Vercel.

**Reply by welschmoor** [reply]
What would it take to make it work for self-hosting? Is it planned? Is it possible? The difference in req/sec between SSR and SSG is ten fold is why I ask. What are the options? Can I by any chance do the same in nginx or caddy? It would be nice if somebody could push me in the right direction. Thank you for answering!!!

**Reply by teemingc** [reply]
It’s not current being worked on but it is planned. You can track the issue here https://github.com/sveltejs/kit/issues/661
